test/request_list.test.js

```diff
@@ -251,6 +251,37 @@ describe('Apify.RequestList', () => {
         mock.restore();
     });
 
+    test('should use the defined proxy server when using `requestsFromUrl`', async () => {
+        const proxyUrls = [
+            'http://proxyurl.usedforthe.download',
+            'http://another.proxy.url',
+        ];
+
+        const mock = sinon.mock(utils.publicUtils);
```

Comment:
dont forget to migrate this to jest when backporting (no need to do it now, we can backport larger chunk later)